.github/workflows/consistency-functional.yml

```diff
@@ -119,16 +121,17 @@ jobs:
           ADMIN_PASSWORD=devstack
           SERVICE_TOKEN=devstack
           DATABASE_PASSWORD=root
+          DATABASE_TYPE=mysql
           HOST_IP=127.0.0.1
           SERVICE_HOST=127.0.0.1
           MYSQL_HOST=127.0.0.1
           RABBIT_HOST=127.0.0.1
           RABBIT_USERID=guest
           RABBIT_PASSWORD=guest
           # Pre-requisite
-          ENABLED_SERVICES=mysql,key
+          ENABLED_SERVICES=key
           # Neutron
-          ENABLED_SERVICES+=,q-svc,q-agt,q-dhcp,q-l3,q-meta,neutron
+          # ENABLED_SERVICES+=,q-svc,q-agt,q-dhcp,q-l3,q-meta,neutron
```

Comment:
oki so if we skip the neutron install the devstack install finish, I need to check why neutron is not starting.